src/components/sections/SociogramPrompts/PromptFieldsEdges.js

```diff
@@ -7,7 +7,7 @@ import { change, Field, formValueSelector } from 'redux-form';
 import * as Fields from '@codaco/ui/lib/components/Fields';
 import { Section, Row } from '@components/EditorLayout';
 import Tip from '../../Tip';
-import { getEdgesForSubject } from './selectors';
+import { getEdgeFilteringWarning, getEdgeFilters, getEdgesForSubject } from './selectors';
```

Comment:
I'm reading this chronologically, but it is suspicious to me that `getEdgeFilterWarning` is being imported from `./selectors` but then isn't used as a selector (via `useSelector`)... 🤔

Reply:
resolved [3422a77](https://github.com/complexdatacollective/Architect/pull/840/commits/3422a77b6b5bc3fa4cfd493966982cf82585eafa)